pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -1184,7 +1187,8 @@ private String setupNewPartitionGroup(TableConfig tableConfig, PartitionLevelStr
 
     CommittingSegmentDescriptor committingSegmentDescriptor = new CommittingSegmentDescriptor(null, startOffset, 0);
     createNewSegmentZKMetadata(tableConfig, streamConfig, newLLCSegmentName, creationTimeMs,
-        committingSegmentDescriptor, null, instancePartitions, numPartitionGroups, numReplicas);
+        committingSegmentDescriptor, null, instancePartitions, numPartitionGroups, numReplicas,
+        Collections.singletonList(partitionGroupMetadata));
```

Comment:
this should pass the whole list right? if we send just the one, every partition will update threshold based on its own partition, and this will not be the same as using only 0th/lowest available partition.

Reply:
+1